modules/engine/src/testing/index.spec.ts

```diff
@@ -69,9 +81,33 @@ describe("VectorEngine", () => {
   // complex logic within the index file of the module)
 
   // NOTE: these are nice-to-haves
+
   describe.skip("setup", () => {
-    it("should fail if it cannot get chainProviders", async () => {});
-    it("should fail if vector.setup fails", async () => {});
+    it("should fail if it cannot get chainProviders", async () => {
+      // chainService.getChainProviders.resolves(Result.fail(new Error("fail")));
+      const engine = await VectorEngine.connect(
+        Sinon.createStubInstance(MemoryMessagingService),
+        Sinon.createStubInstance(MemoryLockService),
+        storeService,
+        getRandomChannelSigner(),
+        Sinon.createStubInstance(VectorChainService),
+        chainAddresses,
+        log,
+      );
+
+      const rpc = constructRpcRequest<"chan_setup">("chan_setup", {
+        counterpartyIdentifier: counterpartyIdentifier,
+        chainId: chainId,
+        timeout: DEFAULT_CHANNEL_TIMEOUT.toString(),
+      });
+      const res = await engine.request(rpc);
+
+      console.log(res.message);
+      expect(res.message).to.contain("fail");
+    });
+    it("should fail if vector.setup fails", async () => {
+      // vectorSetup.setup.resolves(Result.fail(new Error("fail")));
+    });
```

Comment:
This test should either be skipped or implemented, but shouldn't appear to be passing if nothing is happening

Reply:
Skipped, for now, removing commented code too.